README.md

```diff
@@ -66,17 +66,18 @@ This will start a jupyter session, with several example notebooks detailing vari
 
 DeepCell uses `nvidia-docker` and `tensorflow` to enable GPU processing.
 
-### Build a local docker container
+If using GCP, there are pre-built images which come with CUDA, docker, and nvidia-docker pre-installed: 
+https://console.cloud.google.com/marketplace/details/nvidia-ngc-public/nvidia_gpu_cloud_image
```

Comment:
Can you please format the link into the text?  Maybe "there are pre-built images" would be a good place for a link, or the whole thing?  I don't want to see the whole URL.